content/statsServiceMap.Rmd

```diff
@@ -42,6 +42,8 @@ opts_chunk$set(
 
 This script utilizes the new `dataRetrieval` package access to the [USGS Statistics Web Service](http://waterservices.usgs.gov/rest/Statistics-Service.html).  We will be pulling daily mean data using the daily value service in `readNWISdata`, and using the stats service data to put it in the context of the site's history.  Here we are retrieving data for July 12th in the Upper Midwest, where a major storm system had recently passed through. You can modify this script to look at other areas and dates simply by modifying the `states` and `storm.date` objects.
 
+To run this code, we recommend having either `dataRetreival` version 2.5.13 (currently the latest version on CRAN) or version 2.6.1 or later from GRAN. 
+
```

Comment:
I'd probably delete "or later from GRAN" because that is not well documented to the general public.